src/backend/cuda/subconfigure.m4

```diff
@@ -169,6 +169,50 @@ if test "${have_cuda}" = "yes" ; then
                           ],)],[cuda_version=${version}],[])
         if test ! -z ${cuda_version} ; then break ; fi
     done
+    if test ${with_cuda_sm} = *auto* ; then
+        dnl process auto detection
+        PAC_PUSH_FLAG([IFS])
+        IFS=" "
+        AC_MSG_CHECKING([for CUDA compute capability auto detection])
+        AC_LANG_PUSH([C])
+        ac_ext=cu
+        ac_objext=out
+        ac_compile="$NVCC -o conftest.$ac_objext conftest.$ac_ext >&5"
```

Comment:
do we really need to use `nvcc` here or can we use the runtime API from C?

Reply:
It is runtime API, let me try with C compiler.